### DISCUSSION TITLE
peridigm installation

### DISCUSSION THREAD

**Discussion by Bapun-kumar** [peridigm/peridigm#330] [Q&A]
hello,
When I am trying to install peridigm, this error is showing,

:~/peridigm/build$ sudo make install
[sudo] password for bapun:
[  1%] Building CXX object src/CMakeFiles/PeridigmLib.dir/compute/Peridigm_Compute_Acceleration.cpp.o
In file included from /home/bapun/peridigm/src/compute/Peridigm_Compute_Acceleration.hpp:56,
                 from /home/bapun/peridigm/src/compute/Peridigm_Compute_Acceleration.cpp:50:
/home/bapun/peridigm/src/compute/Peridigm_Compute.hpp:51:10: fatal error: Epetra_Comm.h: No such file or directory
   51 | #include <Epetra_Comm.h>
      |          ^~~~~~~~~~~~~~~
compilation terminated.
make[2]: *** [src/CMakeFiles/PeridigmLib.dir/build.make:76: src/CMakeFiles/PeridigmLib.dir/compute/Peridigm_Compute_Acceleration.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:1257: src/CMakeFiles/PeridigmLib.dir/all] Error 2
make: *** [Makefile:146: all] Error 2

please help me to resolve this issue

**Comment by oldninja** [comment]
Do you have Trilinos adequately installed?
